README.md

```diff
@@ -154,7 +160,7 @@ ADMIN_PRIVILEGES=true               # App privileges
 
 | Wazuh version | ODFE    | XPACK  |
 |---------------|---------|--------|
-| v4.3.0        | 1.13.2  | 7.11.2 |
+| v4.3.0        |         | 7.11.2 |
```

Comment:
XPACK is from elastic, in 4.3, we will give support to our own opensearch packages and elastic xpack?